### ISSUE TITLE
Separate checks for scheduled and deadline dates?

### ISSUE THREAD

**Issue by zentiggr** [orgzly/orgzly-android#357]
I have searched for existing issues that may be the same as or related to mine. 👼
I'm submitting a **feature request**.
With the addition of the deadline x date access button, an idea of mine seems logical and easy.  I would like to suggest having separate check buttons for each date.
If both repeats are set, each check would advance only it's own repeat.
If only one repeat is set, that button does a repeat and the other is a no op.
If neither repeat is set, either check box moves to DONE.
I think that is the least surprise version of how it might work, and enables a ton of use cases that i do manually now.
I can try to rough up a PR if it's a bigger change than i think it is...

**Comment by nevenz** [comment]
What about a new "Shift" button in a timestamp dialog? So when you click scheduled or deadline time button and the dialog opens, you'd press "Shift" and it would close the dialog and set new time.

It's two clicks away, but I believe not many people use the times likes this and there's never enough room for more buttons, especially if they are kinda of specific.

**Comment by zentiggr** [comment]
(This is only verifying, not arguing:) Didn't the right swipe buttons just
go from five to four? This would only take the button count back to five.

Aside from that, the Shift buttons would be a fine compromise, i could make
them work very well.

On Jun 30, 2018 4:44 AM, "Neven" <notifications@github.com> wrote:

What about a new "Shift" button in a timestamp dialog? So when you click
scheduled or deadline time button and the dialog opens, you'd press "Shift"
and it would close the dialog and set new time.

It's two clicks away, but I believe not many people use the times likes
this and there's never enough room for more buttons, especially if they are
kinda of specific.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/orgzly/orgzly-android/issues/357#issuecomment-401530273>,
or mute the thread

**Comment by nevenz** [comment]
> Didn't the right swipe buttons just go from five to four? This would only take the button count back to five.

Yes, although there's also a possibility that it becomes confusing that those done/shift buttons are, as they are very similar in function.
